src/packageurl/__init__.py

```diff
@@ -546,4 +544,4 @@ def from_string(cls, purl: str) -> "PackageURL":
             encode=False,
         )
 
-        return PackageURL(type, namespace, name, version, qualifiers, subpath)
+        return cls(type, namespace, name, version, qualifiers, subpath)
```

Comment:
This is an import fix that enables appropriate subclassing, which deserves its own bullet item in the PR description if not the changelog, imo! Thanks!